e2e-network/docker/test-01-simple.sh

```diff
@@ -8,6 +8,10 @@ FABLO_HOME="$TEST_TMP/../../.."
 
 export FABLO_HOME
 
+GATEWAY_CLIENT_DIR="$FABLO_HOME/samples/gateway/node"
+ORG1_PEER0_ENV="$TEST_TMP/fablo-target/fabric-config/connection-profiles/connection-profile-org1-peer0.env"
```

Comment:
TBH I don't like the approach with env file - I think you can define proper variables right before calling the gateway, for instance. It seems more straightforward:

```
(
  cd "$GATEWAY_CLIENT_DIR" && 
    export \
      CHANNEL_NAME="..." \
      PEER_ORG_NAME="..." \
      ...
  node server.js > "$GATEWAY_CLIENT_OUTPUT_FILE" 2>&1
)
```